src/common/components/addressForm/addressForm.tpl.html

```diff
@@ -1,37 +1,39 @@
 <div class="row">
   <div class="col-sm-12">
     <div class="form-group is-required" ng-class="{'has-error': ($ctrl.parentForm.addressCountry | showErrors) || $ctrl.loadingCountriesError}">
-      <label translate>{{'COUNTRY'}}</label>
-      <div class="form-group">
+      <label>
+        <span translate>{{'COUNTRY'}}</span>
         <select class="form-control form-control-subtle"
                 name="addressCountry" required
                 ng-model="$ctrl.address.country"
                 ng-options="v.name as v['display-name'] for v in $ctrl.countries | orderBy:'\'display-name\''"
                 ng-change="$ctrl.refreshRegions($ctrl.address.country)"
                 ng-disabled="$ctrl.addressDisabled">
         </select>
-        <div role="alert" ng-if="$ctrl.loadingCountriesError">
-          <div class="help-block"><translate>{{'COUNTRY_LIST_ERROR'}}</translate>
-            <button id="retryButton1" type="button" class="btn btn-default btn-sm" ng-click="$ctrl.loadCountries()" translate>{{'RETRY'}}</button></div>
-        </div>
-        <div role="alert" ng-messages="$ctrl.parentForm.addressCountry.$error" ng-if="($ctrl.parentForm.addressCountry | showErrors)">
-          <div class="help-block" ng-message="required" translate>{{'COUNTRY_SELECT_ERROR'}}</div>
-        </div>
+      </label>
+      <div role="alert" ng-if="$ctrl.loadingCountriesError">
+        <div class="help-block"><translate>{{'COUNTRY_LIST_ERROR'}}</translate>
+          <button id="retryButton1" type="button" class="btn btn-default btn-sm" ng-click="$ctrl.loadCountries()" translate>{{'RETRY'}}</button></div>
+      </div>
```

Comment:
I'd prefer a bit nicer structure of this HTML:

```
      <div role="alert" ng-if="$ctrl.loadingCountriesError">
        <div class="help-block">
          <translate>{{'COUNTRY_LIST_ERROR'}}</translate>
          <button id="retryButton1" type="button" class="btn btn-default btn-sm" ng-click="$ctrl.loadCountries()" translate>{{'RETRY'}}</button>
        </div>
      </div>
```
Not sure if moving translate to its own line will break the spacing, if that is the case, the closing div should still move to its own line.